src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/LicenseAcceptanceWindow.xaml

```diff
@@ -21,6 +19,7 @@
   Title="{x:Static nuget:Resources.WindowTitle_LicenseAcceptance}"
   d:DesignHeight="300"
   d:DesignWidth="300"
+  Loaded="LicenseAcceptanceWindow_Loaded"
```

Comment:
Need to calculate the dimensions based on the screen, but this is only known after loading.